components/net/fetch/cors_cache.rs

```diff
@@ -112,7 +112,7 @@ pub struct BasicCORSCache(Vec<CORSCacheEntry>);
 fn match_headers(cors_cache: &CORSCacheEntry, cors_req: &CacheRequestDetails) -> bool {
     cors_cache.origin == cors_req.origin &&
         cors_cache.url == cors_req.destination &&
```

Comment:
Great point. Didn't think to question that since the Issue was focused on the later logic. After looking at `CacheRequestDetails`, `destination` is the only `Url` member. Is `destination` used as the "current URL" in here?